rules/import.js

```diff
@@ -65,4 +56,7 @@ module.exports = {
 // If using babel, then be sure to parse the code as ES6.
 if (hasBabel) {
   module.exports.settings['import/parser'] = require.resolve('babel-eslint');
+  module.exports.settings['import/resolver'] = {};
+  module.exports.settings['import/resolver'] =
+    'babel-module';
```

Comment:
Oops. There was a previous example that has it set like:

```javascript
{
  'import/resolver': { 'babel-module': {} }
}
```

Turned out not to be needed. 👍